tests/test_shield_privacy_prefs.py

```diff
@@ -12,16 +13,111 @@ def create_ping_rdd(sc, payload):
     ])
 
 
-def create_row(overrides):
+def create_event_row(overrides):
     keys = ["client_id", "branch", "event", "originDomain", "breakage",
             "notes", "study"]
 
-    overrides['study'] = '@shield-study-privacy'
     overrides['client_id'] = 'aa'
 
     return {key: overrides.get(key, None) for key in keys}
 
 
+def create_state_row(overrides):
+    keys = ["client_id", "branch", "study", "study_state"]
```

Comment:
But I think this might be the right behavior, and I need to change the test to match? I.e., I **WANT** the transform to turn the state ping payload `study_name` into `study` in parquet table so it's consistent with the event pings in parquet.